docs/overview/ss58-address-format.md

```diff
@@ -1,15 +1,80 @@
 ---
 title: SS58 Address Format
 ---
-SS58 is a simple checksummed encoding designed for Substrate addresses. It is based on Bitcoin's Base-58-check format with a few alterations. SS58 addresses are used in the Substrate node itself and serve as a robust default for custom chains. Other address formats may be used in addition to or instead of SS58. Because different chains have different means of identifying accounts, SS58 is designed to be extensible.
+
+SS58 is a simple address format designed for Substrate based chains. There's no problem with using other address formats for a chain, but this serves as a robust default. It is heavily based on Bitcoin's Base-58-check format with a few alterations.
+
+The basic idea is a base-58 encoded value which can identify a specific account on the Substrate chain. Different chains have different means of identifying accounts. SS58 is designed to be extensible for this reason.
 
 ## Basic Format
 
-SS58 addresses are created by performing a [base58 encoding](https://en.wikipedia.org/wiki/Base58) on the concatenation of three fields:
+The basic format conforms to:
+
+```
+base58encode ( concat ( <address-type>, <address>, <checksum> ) )
+```
+
+That is, the concatenated byte series of address type, address and checksum then passed into a base-58 encoder. The `base58encode` function is exactly as [defined](https://en.wikipedia.org/wiki/Base58) in Bitcoin and IPFS, using the same alphabet as both.
+
+## Address Type
+
+The `<address-type>` is one or more bytes that describe the precise format of the following bytes.
+
+Currently, there exist four valid values:
+
+- 00000000b (0) Polkadot Live (SS58 checksum preimage)
+- 00000001b (1) Polkadot Live (AccountId checksum preimage)
+- 00101010b (42) Generic Substrate wildcard (SS58 checksum preimage)
+- 00101011b (43) Generic Substrate wildcard (AccountId checksum preimage)
+- 01000000b - 11111111b (64-255) Reserved for future address format extensions.
```

Comment:
I think the goal of the wiki docs here is to allow new projects to claim an index to be used on their chain. So maybe it makes sense to keep the Substrate based indices on this page, and add a link to the wiki with a reference to say something like "all other indices claimed by the community can be found here" or something